modules/test/playwright/tests/users-admin-web/main/gdpr.spec.ts

```diff
@@ -1963,9 +1963,12 @@ testAdmin(
 		);
 		await userAssociatedDataBlogPage.publishButton.click();
 
-		await productMenuPage.goToMessageBoards();
+		await expect(async () => {
+			await productMenuPage.goToMessageBoards();
```

Comment:
same here if the productMenuPage is not available also from the userAssociatedDataMessageBoardPage